Lesson_7_1.py

```diff
@@ -0,0 +1,36 @@
+# Task 1
+
+class Matrix:
+    b = str()
+
+
+    def __init__(self, l: list):
+        self.l = l
+
+    def __add__(self, other):
+        l_new = []
+        for i in range(len(self.l)):
+            el_new = []
+            for j in range(len(self.l[0])):
+                result = self.l[i][j] + other.l[i][j]
+                el_new.append(result)
+            l_new.append(el_new)
+        return Matrix(l_new)
+
+    def __str__(self):
+        for el in self.l:
+            print(f'{el}')
```

Comment:
надо возвращать строку

Reply:
```suggestion
                def __str__(self):
        matrix = ''
        for el in self.l:
            matrix += f'{el}\n'
        return matrix
```
Вот так изменил метод __str__, теперь всё работает

Reply:
```suggestion
                def __str__(self):
                    matrix = ''
                    for el in self.l:
                        matrix += f'{el}\n'
                    return matrix
```
Изменил метод __str__. Теперь всё работает, как надо